src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import static core.basesyntax.Circle.DEFAULT_RADIUS;
+import static core.basesyntax.Figure.MAX_VALUE;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Color COLOR = Color.WHITE;
```

Comment:
```suggestion
    private static final Color COLOR = Color.WHITE;
```
either rename it to DEFAULT_COLOR or use Color.WHITE directly where need instead of creating additional constant field here